systemtests/system_test.py

```diff
@@ -0,0 +1,248 @@
+import requests
+import pytest
+
+SERVER = "http://0.0.0.0:18420/"
+
+
+@pytest.fixture(scope="class")
+def endpoint_getcredentials(request):
+    response = requests.get(SERVER + "getcredentials")
+    request.cls.response = response
+
+@pytest.mark.usefixtures("endpoint_getcredentials")
+class TestGetcredentialsClass:
+    def test_response_getcredentials(self):
+        #response = endpoint_getcredentials
+        assert self.response.status_code == 200
+
+        # save credentials in global variables
+        authenticatedUser = self.response.json()
+        global client_id
```

Comment:
This is good for now, but fyi for future ref, the best practice would probably be to start a new session in each test. Probably just make a "start_session" function that returns the client_id etc and does stuff

Reply:
instead of using globals a bunch to have a single session do multiple things